ariadne/src/Ariadne/Cardano/Backend.hs

```diff
@@ -69,25 +66,27 @@ runCardanoNode cardanoContextVar diffusionVar commonArgs sendCardanoEvent = do
           cfg <- readLoggerConfig lpConfigPath
           pure $ cfg <> cfgBuilder
       nodeArgs = CLI.NodeArgs { behaviorConfigPath = Nothing }
-      extractionWorker =
-        ( [toAction $ \sendActions -> do
-              ask >>= putMVar cardanoContextVar
-              putMVar diffusionVar sendActions]
-        , mempty )
-  bracket_ setupLoggers removeAllHandlers . logException "ariadne" $ runProduction $ do
-      nodeParams <- CLI.getNodeParams "ariadne" commonArgs nodeArgs
+      extractionWorker diffusion = do
+          ask >>= putMVar cardanoContextVar
+          putMVar diffusionVar diffusion
+  bracket_ setupLoggers removeAllHandlers . logException "ariadne" $ do
+      nodeParams <- usingLoggerName ("ariadne" <> "cardano" <> "init") $
+          CLI.getNodeParams "ariadne" commonArgs nodeArgs
       let vssSK = fromJust $ npUserSecret nodeParams ^. usVss
       let sscParams = CLI.gtSscParams commonArgs vssSK (npBehaviorConfig nodeParams)
-      let workers = updateTriggerWorker
-                 <> extractionWorker
-                 <> statusPollingWorker sendCardanoEvent
+      let workers =
+              [ updateTriggerWorker
+              , extractionWorker
+              , statusPollingWorker sendCardanoEvent
+              ]
       runNodeReal nodeParams sscParams workers
 
-statusPollingWorker
-  :: (HasConfigurations)
-  => (CardanoEvent -> IO ())
-  -> ([WorkerSpec CardanoMode], OutSpecs)
-statusPollingWorker sendCardanoEvent = first pure $ worker mempty $ \_ -> do
+statusPollingWorker ::
+       (HasConfigurations)
```

Comment:
I thought reflection was removed from `cardano-sl/develop`?

Reply:
Not fully.